domains/certificates/Query.API/API.IntegrationTests/Repositories/SyncStateTests.cs

```diff
@@ -25,14 +29,7 @@ public SyncStateTests(MartenDbContainer martenDbContainer)
     [Fact]
     public async Task GetPeriodStartTime_NoDataInStore_ReturnsContractStartDate()
     {
-        CertificateIssuingContract contract = new()
-        {
-            GSRN = "1234",
-            GridArea = "SomeGridArea",
-            MeteringPointType = MeteringPointType.Production,
-            MeteringPointOwner = "SomeMeteringPointOwner",
-            StartDate = DateTimeOffset.Now.AddDays(-1)
-        };
+        var info = syncInfo with { GSRN = "1234" };
```

Comment:
Minor: Gsrn is already set to "1234", but it might be a good idea to be explicit on what we are testing on.

Reply:
I have changed this in 96189a501abaaac94d76d7f58cae520289846965 to use our GSRN Helper class, which can generate random valid GSRN's. Think that makes it clearer that it is just a random GSRN.